src/Components/PokemonCard/PokemonCard.js

```diff
@@ -0,0 +1,41 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import './PokemonCard.css';
+
+
+const PokemonCard =({url}) => {
+    const [pokemonData, setPokemonData] = useState([]);
```

Comment:
Zodra de setter aangeroepen is, dan is `pokemon` niet meer van het type array. Ik zou daarom de default waarde `undefined` maken
```suggestion
    const [pokemonData, setPokemonData] = useState();
```